crates/ruff_python_formatter/src/preview.rs

```diff
@@ -19,6 +19,13 @@ pub(crate) fn is_f_string_formatting_enabled(context: &PyFormatContext) -> bool
     context.is_preview()
 }
 
+/// See [#13539](https://github.com/astral-sh/ruff/pull/13539)
+pub(crate) fn is_f_string_implicit_concatenated_string_literal_quotes_enabled(
```

Comment:
I created a new preview style for now in case we push out the f-string stabilization. We can merge this change with the f-string preview style if we decide to ship both changes.